scripts/generate_instrumentation_bootstrap.py

```diff
@@ -53,12 +53,16 @@
     "bootstrap_gen.py",
 )
 
+packages_to_ignore = ["opentelemetry-instrumentation-aws-lambda"]
```

Comment:
This file is to generate the `bootstrap_gen.py` file used to autoinstrument using `bootstrap.py`. Any reason why we are removing it?

Reply:
Right. 

1. This file is used in `tox -e generate` command to generate the entries in bootstrap_gen.py. 

2. Yes, we are removing it because the LambdaInstrumentor only makes sense when running in Lambda runtime, so the users should opt-in. For example, to use the LambdaInstrumentor you should add and use this [wrapper](https://github.com/open-telemetry/opentelemetry-lambda/blob/main/python/src/otel/otel_sdk/otel_wrapper.py#L52) and this lambda layer cares of installing the package, [see](https://github.com/open-telemetry/opentelemetry-lambda/blob/main/python/src/otel/otel_sdk/requirements.txt#L4).

Options: 
1. (this PR) We can opt for removing it here, so the package won't be installed through bootstrap
2. We can also include the instrumentation-aws-lambda package in `package_to_exclude` in `_load_instrumentors` to not load the LambdaInstrumentor by default. But the question is: does it make sense to have the package installed and not have the respective instrumentor loaded?

Reply:
If we want to be less radical, we can just not load the instrumentor by default but continue to install the package as default through bootstrap. Wdyt? When we had the issue with the aws-lambda the workaround was: remove the package or just disable the instrumentation through environment variable `OTEL_PYTHON_DISABLED_INSTRUMENTATIONS`

Reply:
I think I'm fine with the PR as it is. Maybe jsut add a comment to outline what was discussed here and why we are exlcluding it?